numba/cuda/compiler.py

```diff
@@ -190,15 +190,23 @@ def compile_cuda(pyfunc, return_type, args, debug=False, lineinfo=False,
     flags.no_compile = True
     flags.no_cpython_wrapper = True
     flags.no_cfunc_wrapper = True
+
+    # Both debug and lineinfo turn on debug information in the compiled code,
```

Comment:
Yes, I think so.

Reply:
I decided to go with a warning here rather than raising an error - thinking about it, some people might have both in existing code (especially in test suites). I'll think about upgrading the warning to an exception in a following release rather than breaking things immediately.

Reply:
> I decided to go with a warning here rather than raising an error - thinking about it, some people might have both in existing code (especially in test suites). I'll think about upgrading the warning to an exception in a following release rather than breaking things immediately.

Perhaps open a ticket to track?